src/main/java/de/sakpaas/backend/exception/ApplicationException.java

```diff
@@ -0,0 +1,41 @@
+package de.sakpaas.backend.exception;
+
+import lombok.Getter;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.server.ResponseStatusException;
+
+/**
+ * Exception that will is used by the Expect class in order to throw errors throughout the whole

```

Comment:
> that will is used
-> unschön

Reply:
Gut aufgepasst, hatte ich wohl übersehen, als ich den Kommentar nochmal überarbeitet hab. Danke, behoben

that will is used -> that will be used